agasc/supplement/utils.py

```diff
@@ -73,3 +79,60 @@ def get_supplement_table(name, agasc_dir=None, as_dict=False):
         out = Table(dat)
 
     return out
+
+
+def _load_or_create(filename, table_name):
+    try:
+        table = Table.read(filename, format='hdf5', path=table_name)
+    except OSError as e:
+        logger.debug(f'Creating agasc supplement table "{table_name}"')
+        if not filename.exists() or str(e) == f'Path {table_name} does not exist':
+            table = Table()
+        else:
+            raise
+    return table
+
+
+def save_version(filename, **kwargs):
+    """Save the version of a supplement table to the "versions" table.
+
+    Along with the version, the time of update is also added to another table called "last_updated"
+
+    Example usage::
+
+        from agasc.supplement.utils import save_version
+        save_version('agasc_supplement.h5', mags='4.10.2')
+
+    The different versions can be retrieved from the default supplement::
+
+        from agasc.supplement.utils import get_supplement_table
+        versions = get_supplement_table('versions')
+
+    :param filename: pathlib.Path
+    :param kwargs: dict
```

Comment:
Since this function is always being called (except in testing) with `save_version(filename, <table_name>=agasc.__version__)`, maybe just simplify to `save_version(filename, table_name)`?

Reply:
yes, I thought of that, and I also though of having a default argument. I can remove the argument because it makes it simpler, and one can always add it if you want to use a custom version string.

Are you sure you will not want a custom version string? I changed it according to the suggestion, but one can still undo that commit.